scripts/start-eigenda-proxy.sh

```diff
@@ -1,21 +1,31 @@
 #!/usr/bin/env bash
 
-echo "Pull eigenda-proxy container"
-docker pull  ghcr.io/layr-labs/eigenda-proxy@sha256:10a4762f5c43e9037835617e6ec0b03da34012df87048a363f43b969ab93679b
+echo "==== Pull eigenda-proxy container ===="
+docker pull  ghcr.io/layr-labs/eigenda-proxy:v1.6.0
 
-echo "Tagging image"
-docker tag  ghcr.io/layr-labs/eigenda-proxy@sha256:10a4762f5c43e9037835617e6ec0b03da34012df87048a363f43b969ab93679b eigenda-proxy-nitro-test
+echo "==== Starting eigenda-proxy container ===="
 
-echo "Start eigenda-proxy container"
+# proxy has a bug currently which forces the use of the service manager address 
+# & eth rpc despite cert verification being disabled.
 
-docker run -d --name eigenda-proxy-nitro-test \
+docker run -d --name eigenda-proxy-nitro-test-instance \
   -p 4242:6666 \
   -e EIGENDA_PROXY_ADDR=0.0.0.0 \
   -e EIGENDA_PROXY_PORT=6666 \
-  -e MEMSTORE_ENABLED=true \
-  -e MEMSTORE_EXPIRATION=1m \
-  -e EIGENDA_PROXY_TARGET_URL=http://localhost:3000 \
-  eigenda-proxy-nitro-test
+  -e EIGENDA_PROXY_MEMSTORE_ENABLED=true \
+  -e EIGENDA_PROXY_MEMSTORE_EXPIRATION=1m \
+  -e EIGENDA_PROXY_EIGENDA_ETH_RPC=http://localhost:6969 \
+  -e EIGENDA_PROXY_EIGENDA_SERVICE_MANAGER_ADDR="0x0000000000000000000000000000000000000000" \
+  -e EIGENDA_PROXY_EIGENDA_CERT_VERIFICATION_DISABLED=true \
+  ghcr.io/layr-labs/eigenda-proxy:v1.6.0
+
+# shellcheck disable=SC2181
+if [ $? -ne 0 ]; then
+  echo "==== Failed to start eigenda-proxy container ===="
+  exit 1
+fi
```

Comment:
nit: this will catch if docker didn't manager to start the process, but won't catch if the process is started but crashed on startup.